src/platform_impl/android/mod.rs

```diff
@@ -239,9 +239,26 @@ impl<T: 'static> EventLoop<T> {
                                             }
                                         }
                                     }
-                                    InputEvent::KeyEvent(_) => {
-                                        // TODO
-                                        handled = false;
+                                    InputEvent::KeyEvent(key) => {
+                                        let event = event::Event::WindowEvent {
+                                            window_id,
+                                            event: event::WindowEvent::KeyboardInput {
+                                                device_id,
+                                                input: event::KeyboardInput {
+                                                    scancode: key.key_code() as u32,
+                                                    state: event::ElementState::Released,
```

Comment:
The android [`key action`](https://docs.rs/ndk/0.3.0/ndk/event/struct.KeyEvent.html#method.action) should be used for mapping to the element state, otherwise Released will be reported twice as it the attached logs.

Reply:
Done